PatientRegistrationApp/src/components/SinglePatientFollowUp.vue

```diff
@@ -89,41 +56,31 @@ export default {
 				givenName: '',
         suffix: ''
       },
-      qrText: '',
-      qrSrc: null
+      correctionLevel: "H"
     };
   },
   beforeDestroy (){
   this.reset()
   },
   methods: {
     updatePersonalInfoData(personalInfoPayload) {
-      this.dataPersonalInfo = personalInfoPayload
-    },
-    createObjectUrl (err, canvas) {
+      this.dataPersonalInfo = personalInfoPayload;
+      this.qrValue = this.dataPersonalInfo.familyName + ", " + this.dataPersonalInfo.givenName + " " + this.dataPersonalInfo.middleName + " " + this.dataPersonalInfo.suffix
+    }
+    /*createObjectUrl (err, canvas) {
```

Comment:
Do we still need this stuff? If so, I'd add a `// TODO explain what needs to be done` or remove the code if not.